.config/gitpod/Dockerfile

```diff
@@ -36,6 +40,12 @@ RUN PYTHON_VERSION=$(cat /tmp/python_version.config | cut -d '=' -f 2) && \
     $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
     && apt-get update \
     && apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io \
+    ## Install npm
+    && curl -fsSL https://deb.nodesource.com/setup_20.x |bash - \
+    && apt-get install  -y --no-install-recommends nodejs \
+    && npm install -g npm@9.6.7 \
+    && npm install -g \
+        ungit@1.5.23 \
```

Comment:
> npm@9.6.7
> ungit@1.5.23

Please consider specifying versions in variables.

Reply:
Regarding the versioning of npm and ungit, I agree with your suggestion to specify the versions in variables. This would indeed make the Dockerfile more maintainable. I will make the necessary changes to the pull request.

Reply:
Done https://github.com/vitabaks/postgresql_cluster/pull/352/commits/26346dda27774e046cadff048a3659b12ec11363